place_tracker/ios/.gitignore

```diff
@@ -25,6 +25,7 @@ Flutter/flutter_assets/
 Flutter/flutter_export_environment.sh
 ServiceDefinitions.json
 Runner/GeneratedPluginRegistrant.*
+build/
```

Comment:
I've been putting this in as `/build/` to make sure it's only the generated directory